.github/workflows/build.yml

```diff
@@ -1,61 +1,62 @@
-name: "Build"
+name: Build
 
 on:
   - push
   - pull_request
 
 jobs:
-  build: # Check build on various OSes
-
+  build:
     strategy:
+      fail-fast: false
       matrix:
         os:
           - ubuntu-latest
+          - macos-latest
+          - windows-latest
         ocaml-compiler:
-          - 4.02.x
-          - 4.04.x
-          - 4.06.x
-          - 4.08.x
-          - 4.10.x
-          - 4.12.x
-          - 4.14.x
-          - 5.0.x
-          - 5.1.x
+          - 4
+          - 5
```

Comment:
If I understand correctly, this runs odoc's test in 4.02 on ubuntu latest (due to the `include` below), and on 4.14 and the latest 5 version for linux, windows and mac.

I think it makes sense for odoc to continue checking each individual versions (4.07, 5.0 and 5.1, ...). Indeed, odoc has a lot of compatibility code for instance for the different ASTs or for the shapes, that can easily break so that we want to continue testing. (This compatibility code might work for 4.02 and 4.14 but not for an intermediate versions).

Reply:
Reverted